simple/cq_data.c

```diff
@@ -481,7 +481,7 @@ int main(int argc, char **argv)
 
 	hints->ep_attr->type = FI_EP_MSG;
 	hints->caps = FI_MSG;
-	hints->mode = FI_LOCAL_MR | FI_PROV_MR_ATTR;
+	hints->mode = FI_LOCAL_MR | FI_PROV_MR_ATTR | FI_RX_CQ_DATA;
```

Comment:
FI_RX_CQ_DATA is defined as a mode bit in fi_getinfo manpage. Does the manpage need to be corrected? The current verbs provider code checks for this mode bit if the app sets the cq_data_size domain attribute.

Btw, since cq_data transferred from a remote endpoint consumes a posted recv and an app might not know if it's going to receive remote cq data, shouldn't FI_RX_CQ_DATA be a mode bit and always required to be set by the app?